src/rust/engine/stdio/src/lib.rs

```diff
@@ -417,6 +417,9 @@ thread_local! {
   static THREAD_DESTINATION: RefCell<Arc<Destination>> = RefCell::new(Arc::new(Destination(Mutex::new(InnerDestination::Logging))))
 }
 
+// Note: The behavior of this task_local! invocation is affected by the `tokio_no_const_thread_local`
+// config set in `src/rust/engine/.cargo/config`. Without that config, this item triggers the
+// `clippy::declare_interior_mutable_const` lint.
```

Comment:
@stuhood: FYI I had to set `--cfg tokio_no_const_thread_local` to avoid the [`clippy::declare_interior_mutable_const`](https://rust-lang.github.io/rust-clippy/master/index.html#declare_interior_mutable_const) lint. This is related to the tokio upgrade contained in this PR.

Reply:
Apparently the lint should go away on Rust v1.64.0. This will land Rust 1.64.0 upgrade (https://github.com/pantsbuild/pants/pull/17031), then tokio upgrade, then this PR. <sigh>